app/src/main/java/org/fossasia/pslab/communication/ScienceLab.java

```diff
@@ -84,7 +85,7 @@ public ScienceLab(CommunicationHandler communicationHandler) {
         if (isDeviceFound() && MainActivity.hasPermission) {
             try {
                 mCommunicationHandler.open();
-                //SystemClock.sleep(200);
+                //Thread.sleep(200);
```

Comment:
Good work @viveksb007 ! Thanks for the PR! I guess this was for testing :) We can remove this when merging! :+1: